src/actions/action.js

```diff
@@ -45,7 +45,10 @@ const getActions = async (
     }
 
     // create url from params
-    const url = `//${apiHost + ACTIONS_ENDPOINT}?${Qs.stringify(params)}`;
+    const url = `//${apiHost + ACTIONS_ENDPOINT}?${Qs.stringify({
```

Comment:
I think this is wrong. We should probably be pushing the current space into the spaceId array of params. Does that make sense?